Dockerfile

```diff
@@ -1,23 +1,5 @@
-FROM navikt/node-express:1.0.0
+FROM navikt/pus-decorator:216.20190522.1711
```

Comment:
Bruke siste versjon, eller er det for vågalt? 
228.20190926.1521

Reply:
Nei, jeg tester med siste :D

Reply:
Looking good, @ma10s ! Bra spotta 👍